pontoon/settings/base.py

```diff
@@ -763,7 +763,7 @@ def _allowed_hosts():
 # For the sake of integration with other sites,
 # some of javascript files (e.g. pontoon.js)
```

Comment:
@PFischbeck Great work!
@mathjazz I think that was required by the legacy Translation view and the in-place editor. Does any Pontoon feature still needs it?

Reply:
A line under hints at GraphQL.